pydicom/_version.py

```diff
@@ -1,3 +1,4 @@
 """Pure python package for DICOM medical file reading and writing."""
 
 __version__ = '1.1.0.dev0'
+__version_info__ = tuple(__version__.split('.')[:3])
```

Comment:
This won't work in all cases, e.g. like the '1.0.1rc1' I've been working with in 1.0.X branch, which is compliant with [Python examples](https://packaging.python.org/tutorials/distributing-packages/#choosing-a-versioning-scheme)

Reply:
Everything else looks good.  Just that third part of the tuple should also have first non-numeric character stripped to end of string.

Reply:
So could we you a regex, something like r'(\d+\.\d+\.\d+).*' ?

Reply:
I was thinking just to regex the third part, to make it easier.  But if do full one, then I think your example needs to escape the "."'s right?

Reply:
It actually did that - the markdown syntax got in the way...

Reply:
Ah, then that looks like it should handle the versions we would use

Reply:
sorry, don't have the time to work on this at the moment

Reply:
no problem, master is in slow mode now while the release goes out from 1.0.X

Reply:
I added the regex part and backed it out again, as it broke the test (shall have another look).

Reply:
ugh, yes, of course - we still need the tuple, not the string